lib/handler/compress.c

```diff
@@ -66,7 +66,7 @@ static void on_setup_ostream(h2o_filter_t *_self, h2o_req_t *req, h2o_ostream_t
     if (!req->res.mime_attr->is_compressible)
         goto Next;
     /* 100 is a rough estimate */
-    if (req->res.content_length <= 100)
+    if (req->res.content_length <= self->args.min_size)
```

Comment:
Should we better change the operator to `<` since the variable defines the minimum size that gets compressed?